scripts/e2e_setup_cluster.sh

```diff
@@ -126,6 +131,8 @@ echo "## generating K8 api flags files"
 generate_k8_api_data
 echo "## start Kind cluster with precreated CA key/cert"
 create_cluster
```

Comment:
Can we delete cluster before trying to create it ? Might be good to remove any stale clusters on self hosted
```
kind delete cluster
```
It is idempotent.

Reply:
It is already done within create_cluster function. Last time you have suggest that due to observed failures in Kind creation. In result create process should be retried.

line 105
```
 # deploy cluster with kind
  retry kind delete cluster && kind create cluster --config="${PWD}"/kindConfig.yaml
```